reading-notes-intro-computation-programming-python/chapter-one_getting-started/quiz.md

```diff
@@ -3,19 +3,28 @@
 ## What is a computer ?
 
 What does `computer` mean ? 
+> A machine that performs calculations and remembers the results of the calculations, very quickly. Those calculations are executed following a program, a finite sequence or list of instructions.
+
 You can use Wikipedia to find out.
 Find also a usage related to Easter. 
+> Something related to Blaise Pascal, so I would say the Pascaline, the first known calculator.
```

Comment:
Nope. AFAIK, Blaise Pascal was not involved with Easter.

The answer was [computus](https://en.wikipedia.org/wiki/Date_of_Easter), which is an algorithm to find the date of easter.